string markerFile
string objectFile
string imgDir
---
